src/app/platform/components/sidebar.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { Library, Plus } from "lucide-react";
+import LogoMark from "./navbar.svg";
+import Image from "next/image";
+const sidebarItems = [
```

Comment:
Sidebar items should be props to the component because it will depend on which page it is rendered. For example: `API Keys` or some other page developed in future.